specification/communication/data-plane/JobRouter/preview/2021-10-20-preview2/communicationservicejobrouter.json

```diff
@@ -0,0 +1,3635 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Azure Communication Routing Service",
+    "description": "Azure Communication Routing Service",
+    "version": "2021-10-20-preview2"
+  },
+  "paths": {
+    "/routing/classificationPolicies": {
+      "post": {
+        "tags": [
+          "ClassificationPolicies"
+        ],
+        "summary": "Creates a new classification policy.",
+        "operationId": "JobRouter_CreateClassificationPolicyV2",
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "in": "header",
+            "name": "repeatability-request-id",
+            "type": "string"
+          },
+          {
+            "in": "header",
+            "name": "repeatability-first-sent",
+            "type": "string"
+          },
+          {
+            "in": "body",
+            "name": "classificationPolicy",
+            "description": "Model of classification policy to create",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/ClassificationPolicy"
+            }
+          }
+        ],
+        "responses": {
```

Comment:
There are a number of operations in your OpenAPI document that do not have a `default response`. You should make a point to add these. One easy way to quickly identify these is to check out a linter we are experimenting with, Spectral. You can find more information on the [Azure REST API Style Guidelines](https://github.com/Azure/azure-api-style-guide) page.

Reply:
Tracked using [this issue](https://skype.visualstudio.com/SPOOL/_workitems/edit/2784920).